Octokit/Models/Response/CommitCommentReaction.cs

```diff
@@ -0,0 +1,61 @@
+﻿using Octokit.Internal;
+using System;
+using System.Diagnostics;
+using System.Globalization;
+
+namespace Octokit
+{
+    public enum Reaction
+    {
+        [Parameter(Value = "plus_one")]
+        Plus1 = 0,
+        [Parameter(Value = "minus_one")]
+        Minus1 = 1,
+        [Parameter(Value = "laugh")]
+        Laugh = 2,
+        [Parameter(Value = "confused")]
+        Confused = 3,
+        [Parameter(Value = "heart")]
+        Heart = 4,
+        [Parameter(Value = "hooray")]
+        Hooray = 5
+    }
+
+    [DebuggerDisplay("{DebuggerDisplay,nq}")]
+    public class CommitCommentReaction
```

Comment:
This response object is returned for creating reactions on CommitComments, Issues, IssueComments and PullRequestReviewComments, so it should probably be called something more generic like `Reaction` rather than `CommitCommentReaction`